lib/views/index.erb

```diff
@@ -1,22 +1,23 @@
 <!DOCTYPE html>
 <html>
   <h1>Current Board State:</h1>
+    <link rel="stylesheet" href="/styles.css"/>
   <p><%= grid %></p>
   <table>
     <tr>
-      <td><form method="post" action="/make-move/0"><button/></form></td>
-      <td><form method="post" action="/make-move/1"><button/></form></td>
-      <td><form method="post" action="/make-move/2"><button/></form></td>
+      <td><form method="post" action="/make-move/0"><button><%= grid[0] %></button></form></td>
+      <td><form method="post" action="/make-move/1"><button><%= grid[1] %></button></form></td>
```

Comment:
Is there a way to refactor the form method tag? Because It looks like a repetition of a lot of form tags.  Is this how it should be done in this situation?

Reply:
We can't seem to find a way to reduce repetition without using Javascript. which we haven't used at all.